test/graticule/geocoder/mapbox_test.rb

```diff
@@ -28,16 +24,22 @@ def test_locate_success
       end
 
       def test_locate_not_found
-        URI::HTTP.responses << response("mapbox", "empty_results", "json")
+        return unless prepare_response(:empty_results)
```

Comment:
What is the point of the `return` here? Are we ever expecting `prepare_response` to return anything other than a truthy value? And if it didn't, would that pass the test as a false positive as it's written here?

Reply:
Eh, I wasn't sure either but kept went with it after seeing the convention in a number of other test files.